src/metabase/driver/sql/query_processor.clj

```diff
@@ -267,13 +271,24 @@
   [driver [_ _ expression-definition]]
   (->honeysql driver expression-definition))
 
+(defn unix-timestamp->honeysql*
+  "Function in front of `unix-timestamp->honeysql` to translate the special type to the appropriate magnitude (seconds,
```

Comment:
Minor nitpick: in the rest of the code we tend to use `unit` instead of `magnitude`